### DISCUSSION TITLE
有什么优雅的在弹出层和父页面之前传递数据的方法吗？

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2109] [讨论]
目前我倒是可以实现父子层之间传数据，但总是感觉好繁琐，一层层的回调没完没了，有没有什么简洁优雅的方法

**Comment by LSL1618** [comment]
我一般这样传递，父页面定义`window.pageDialog = {};`，子页面取用`window.parent.pageDialog;`。
